src/TestFramework/MSTest.Core/Resources/FrameworkMessages.resx

```diff
@@ -283,4 +283,7 @@ Stack Trace: {4}</value>
   <data name="DynamicDataValueNull" xml:space="preserve">
     <value>Value returned by property or method {0} shouldn't be null.</value>
   </data>
+  <data name="DynamicDataDisplayName" xml:space="preserve">
```

Comment:
Please run "build.cmd -uxlf" and check-in all the files with localization changes.

Reply:
I ran this and have committed the xlf files. I'm assuming someone or some magic will do the translation?

Reply:
Yes please. We just needed modified xlf files. We will take care of the translation 😄